overrides/config-overrides/expert/betterquesting/DefaultQuests.json

```diff
@@ -37137,7 +37137,11 @@
               "Count:3": 3,
               "Damage:2": 13,
               "OreDict:8": "",
-              "id:8": "gregtech:meta_block_compressed_2006"
+              "id:8": "betterquesting:placeholder",
+              "tag:10": {
+                "orig_id:8": "gregtech:meta_block_compressed_2006",
+                "orig_meta:3": 13
+              }
```

Comment:
I'm not sure what the deal is here--the couple quests that got replaces with placeholders like this don't seem to actually be in the questbook.

Reply:
This is because those are also broken. You need to replace those blocks with the new blocks.

The new blocks begin with 'nomilabs:meta_block_compressed_'.
This needs to be done for each meta block compressed with the last value > 2000.

Thanks in advance.